src/_includes/v2/footer/nav-links.html

```diff
@@ -0,0 +1,6 @@
+<p><a class="font-weight-bold" href>Data plans</a></p>
+<p><a class="font-weight-bold" href>Route schedule</a></p>
+<!-- no hyphen -->
+<p><a class="font-weight-bold" href>GTFS Realtime</a></p>
```

Comment:
In the figma design this is `GTFS-Realtime`. i opted for consistency with the header.

https://www.figma.com/design/TfWk1iDHdlt7bSA3DjQuQv?node-id=8802-11904&m=dev#1558145142